### ISSUE TITLE
Skipping branch status checks

### ISSUE THREAD

**Issue by jdbence** [tool3/bump#4]
Hello! I would like to enable Github branch protection rules on my master branch. Is it possible to update this action to skip status checks when making a commit to increase the package version?

I'm guessing it has to be added here
https://github.com/tool3/bump/blob/876e407fd016a07a08b5ff1dde38de402b0380fc/index.js#L48

Github's example
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks#skipping-and-requesting-checks-for-individual-commits
```JS
$ git commit -m "Update README.
>
>
skip-checks: true
```

**Comment by tool3** [comment]
hey @jdbence  ! 

absolutely.. can you run a test against branch `feat/skip_checks`? 

so just run a workflow using:
```yml
- uses: tool3/bump@feat/skip_checks
```

**Comment by jdbence** [comment]
It doesn't seem to work. It might be from the ``` %s``` at the end of the literal or that you need another ```\n``` to actually have two spaces between the ```skip-checks: true```
https://github.com/tool3/bump/blob/3febd9f54c9e02e9916e0dd17bbd6da7c68845f1/index.js#L53

**Comment by tool3** [comment]
yeah just noticed that... I fixed it for now by putting the version string at the beginning of the commit msg

so please try again :)

**Comment by jdbence** [comment]
I get this same error. Did you see my edited message above about adding another ```\n```?

```
remote: error: GH006: Protected branch update failed for refs/heads/master.        
remote: error: Required status check "Test" is expected. 
```

**Comment by tool3** [comment]
oh no I did't sorry about that...just added another `\n`

**Comment by jdbence** [comment]
That seems to have the same result. Maybe its from not having a starting quote for the message?
```
After your commit description, instead of a closing quotation, add two empty lines followed by skip-checks: true:
```

**Comment by tool3** [comment]
Yeah I was just able to push a real commit with this format so let me try another thing

**Comment by tool3** [comment]
this is quite a unique requirement since I am running all of this through `npm version`

so I just added `>` after each line break to see if git parses that correctly

this is probably not going to work too... so let me think of a creative solution

of course you are more than welcome to have a go yourself

**Comment by jdbence** [comment]
https://github.com/tool3/bump/blob/144b99f89e529d13de3c35f74d6931fad917aef8/index.js#L34

What if you create them with actual spaces?
```JS
commitMessage = '`${commitMessage}


skip-checks: true'
```

**Comment by tool3** [comment]
yup.. just did buddy

**Comment by tool3** [comment]
@jdbence any luck? sorry had very little time to work on this today

**Comment by jdbence** [comment]
I tried and bunch of different things. I contacted Github support to see if they have an idea. I'll let you know if/what they say.

**Comment by jdbence** [comment]
I got a reply from a Github Rep.
They said adding the '[skip ci]' to the message should be enough but you cant read the message directly anymore.
They suggested using another Github Action in combination with this one that checks for the 'skip-checks' we set
[https://github.com/marketplace?type=actions&query=commit+message](https://github.com/marketplace?type=actions&query=commit+message)

So I'll see if I can get something like that working.

**Comment by tool3** [comment]
@jdbence thanks! Please update if you find anything.
A PR is more than welcome 😄
